### ISSUE TITLE
Perinuclear cardiomyocyte vacuolization

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7086]
**Preferred term label:**
Perinuclear cardiomyocyte vacuolization

**Synonyms**
Perinuclear vacuolization of cardiomyocytes

**Definition (free text, please give PubMed ID)**
Formation of vacuoles, i.e.,  membrane-bound organelles, in the portion of the cytoplasm near the nucleus within myocytes of the heart. 

**Parent term (use hpo.jax.org/app)**
Abnormal myocardium morphology HP:0001637

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619371 

2/2  PMID: 32514796

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also, cardiomyocyte hypertrophy
def:  increase in cell size, enhanced protein synthesis, and heightened organization of the sarcomere.  PMID:15066961

**Comment by pnrobinson** [comment]
added
